src/main/fractions.ts

```diff
@@ -23,7 +23,7 @@ export class FractionsConstraint implements ValidatorConstraintInterface {
       throw new Error('Maximum allowed decimal places has to be specified and positive value')
     }
 
-    if (value === undefined) {
+    if (value === undefined || typeof value !== 'number') {
```

Comment:
This is technical debt at this point. A citizen frontend does not require it as it solved the problem in better way. A legal frontend in turn has more naive implementation. Refactoring legal frontend would be a challenge now so I am happy to accept this technical debt for now. I will create story for future refactoring of the legal app.